docs/source/examples/simple-landscape-qiskit.myst

```diff
@@ -0,0 +1,249 @@
+    circuit.rx(gamma, 0)
+    
+    return circuit
+```
+
+We can visualize the circuit for a particular $\gamma$ as follows.
+
+```{code-cell} ipython3
+circuit = variational_circuit(gamma=np.pi)
+circuit.draw()
+```
+
+## Defining the executor functions with and without noise
+To use error mitigation methods in Mitiq, we define an executor function which computes the expectation value of a simple Hamiltonian $H=Z \otimes Z$, i.e., Pauli-$Z$ on each qubit. To compare to the noiseless result, we define both a noiseless and a noisy executor below.
+
+```{code-cell} ipython3
+# observable to measure
+z = np.diag([1, -1])
+hamiltonian = np.kron(z, z)
+
+def noiseless_executor(circuit: QuantumCircuit) -> float:
+    """Simulates the execution of a circuit without noise.
+
+    Args:
+        circuit: The input circuit.
+
+    Returns:
+        The expectation value of the ZZ observable.
+    """
+    # avoid mutating the input circuit
+    circ = circuit.copy()
+    circ.save_density_matrix()
+
+    # execute experiment without noise
+    job = qiskit.execute(
+        experiments=circ,
+        backend=qiskit.Aer.get_backend("aer_simulator_density_matrix"),
+        noise_model=None,
+        # we want all gates to be actually applied,
+        # so we skip any circuit optimization 
+        optimization_level=0,
+        shots=1,
+    )
+    rho = job.result().data()["density_matrix"]
+
+    expectation = np.real(np.trace(rho @ hamiltonian))
+    return expectation 
+    
+
+
+# strength of noise channel
+noise_level = 0.04
+
+def executor_with_noise(circuit: QuantumCircuit) -> float:
+    """Simulates the execution of a circuit with depolarizing noise.
+
+    Args:
+        circuit: The input circuit.
+
+    Returns:
+        The expectation value of the ZZ hamiltonian.
+    """
+    # avoid mutating the input circuit
+    circ = circuit.copy()
+    circ.save_density_matrix()
+    
+    # Initialize qiskit noise model. In this case a depolarizing
+    # noise model with the same noise strength on all gates
+    noise_model = NoiseModel()
+    noise_model.add_all_qubit_quantum_error(
+        depolarizing_error(noise_level, 1), ["rx"]
+    )
+    noise_model.add_all_qubit_quantum_error(
+        depolarizing_error(noise_level, 2), ["cx"]
+    ) 
+    
+
+    # execute experiment with depolarizing noise
+    job = qiskit.execute(
+        experiments=circ,
+        backend=qiskit.Aer.get_backend("aer_simulator_density_matrix"),
+        noise_model=noise_model,
+        basis_gates=noise_model.basis_gates + ["save_density_matrix"],
+        # we want all gates to be actually applied,
+        # so we skip any circuit optimization 
+        optimization_level=0,
+        shots=1,
+    )
+    rho = job.result().data()["density_matrix"]
+
+    expectation = np.real(np.trace(rho @ hamiltonian))
+    return expectation 
+```
+
+**Note:** The above code block uses depolarizing noise, but any Qiskit `NoiseModel` can be substituted in.
```

Comment:
This (and the one on line 162) might benefit from using the `note` directive. ([more info](https://myst-parser.readthedocs.io/en/latest/syntax/roles-and-directives.html#admonitions))